src/nonce.iced

```diff
@@ -0,0 +1,18 @@
+nonceBytes = 24
+sigNonceBytes = 16
+{prng} = require('crypto')
+uint64be = require('uint64be')
+
+exports.nonceForSenderKeySecretBox = () -> return new Buffer('saltpack_sender_key_sbox')
```

Comment:
Move these to the saltpack library, since they're too specific to saltpack for inclusion here.